clustering/infinispan/extension/src/main/resources/org/jboss/as/clustering/infinispan/subsystem/LocalDescriptions.properties

```diff
@@ -157,9 +157,8 @@ infinispan.component.partition-handling.force-available=Forces a cache with degr
 infinispan.component.state-transfer=The state transfer configuration for distributed and replicated caches.
 infinispan.component.state-transfer.add=Add a state transfer configuration.
 infinispan.component.state-transfer.remove=Remove a state transfer configuration.
-infinispan.component.state-transfer.enabled=If enabled, this will cause the cache to ask neighboring caches for state when it starts up, so the cache starts 'warm', although it will impact startup time.
-infinispan.component.state-transfer.enabled.deprecated=Deprecated. Always enabled for replicated and distributed caches.
-infinispan.component.state-transfer.timeout=The maximum amount of time (ms) to wait for state from neighboring caches, before throwing an exception and aborting startup.
+infinispan.component.state-transfer.enabled=Enables initial state transfer. Disabling initial state transfer can leave cache vulnerable to data-loss, especially when using distribution mode with a minimal number of owners.
+infinispan.component.state-transfer.timeout=The maximum amount of time (ms) to wait for state from neighboring caches, before throwing an exception and aborting startup. A timeout of 0 means the cache will be available immediately after joining, and initial state transfer is non-blocking.
```

Comment:
Does `timeout` support negative values (e.g. to wait forever)?
A validator could constrain the attribute to only accepted values (>= 0 aiui).

Reply:
I'll clarify the description.  Essentially, anything > 0 is treated as a timeout.  Anything else, i.e. 0 or negative, uses non-blocking asynchronous state transfer.  Thus, no validator is needed.